packages/settings-network/settings-network.spec

```diff
@@ -0,0 +1,39 @@
+%global _cross_first_party 1
+%undefine _debugsource_packages
+
+%global extension_name network
+
+Name: %{_cross_os}settings-%{extension_name}
+Version: 0.0
+Release: 0%{?dist}
+Summary: settings-%{extension_name}
+License: Apache-2.0 OR MIT
+URL: https://github.com/bottlerocket-os/bottlerocket
+
+BuildRequires: %{_cross_os}glibc-devel
```

Comment:
Likewise, we want to add a `Requires: %{_cross_os}netdog`

This symbol would only be available once we merge in https://github.com/bottlerocket-os/bottlerocket/pull/3700

Reply:
> Likewise, we want to add a Requires: %{_cross_os}netdog
>
> This symbol would only be available once we merge in https://github.com/bottlerocket-os/bottlerocket/pull/3700

Isn't it the other way around? Doesn't `netdog` depend on its settings extension?

Edit: oh I see, this extension calls out to `netdog` makes sense. This is a setting that relies on netdog to find its value.